code_survey/controllers/main.py

```diff
@@ -19,7 +19,7 @@ def survey_submit(self, survey_token, answer_token, **post):
         if access_data['validity_code'] is not True:
```

Comment:
```suggestion
        if not ['validity_code']:
```
This could achieve the same :) and looks cleaner, however test it out just to be sure :+1:

Reply:
This code comes from standard, as we didn't find a way to inherit the parent method and still add the change we needed